### PR TITLE
support Vary Toy infer

### PR DESCRIPTION
**Pull request by panshaowu**
Thank you for your contribution to the MindOCR repo.
Before submitting this PR, please make sure:

- [ ] You have read the [Contributing Guidelines on pull requests](https://github.com/mindspore-lab/mindcv/blob/main/CONTRIBUTING.md)
- [ ] Your code builds clean without any errors or warnings
- [ ] You are using approved terminology
- [ ] You have added unit tests

## Motivation

Add infer support of OCR LLM Vary and Vary Toy. 

## Test Plan

(How should this PR be tested? Do you require special setup to run the test or repro the fixed bug?)

## Related Issues and PRs

(Is this PR part of a group of changes? Link the other relevant PRs and Issues here. Use https://help.github.com/en/articles/closing-issues-using-keywords for help on GitHub syntax)

### CODE REVIEW

File: mindocr/models/utils/loss.py
Comment (panshaowu) [inline mindocr/models/utils/loss.py:1]: 建议审视该文件内的代码可否复用已有代码

**Review by chengtianwu — APPROVED** [review]

File: configs/llm/vary/README.md [hunk 1]
```diff
@@ -0,0 +1,111 @@
...
+### 3.1 Environment and Model Preparation
+
+#### 3.1.1 Installation
+
+For details about environment installation, see MindOCR [installation instruction](https://github.com/mindspore-lab/mindocr#installation).
+
+In addition, you need to install the `tiktoken` using the following shell command:
+
+``` shell
+pip install tiktoken
```
Comment (tonytonglt) [inline configs/llm/vary/README.md:36]: 可否加入requirements.txt？

File: configs/llm/vary/README.md [hunk 2]
```diff
@@ -0,0 +1,111 @@
...
+  ...
+  tokenizer:
+    vocab_file: "/path/to/qwen.tiktoken"  # Path of the tokenizer
+  ...
+...
+```
+
+#### 3.1.3 Model Preparation
+
+Users can download Vary-toy's [weight](https://huggingface.co/HaoranWei/Vary-toy/blob/main/pytorch_model.bin), and then perform the weight conversion according to the following steps.
```
Comment (tonytonglt) [inline configs/llm/vary/README.md:63]: 是否可以直接提供转换好的weights文件供用户下载使用？

Comment (chengtianwu) [reply configs/llm/vary/README.md:63]: 可以

File: configs/llm/vary/README.md (hunk 1 above)
Comment (panshaowu) [reply configs/llm/vary/README.md:36]: 需要添加说明“You shall upgrade your python to 3.8 or above version, if you want to experience Vary-toy.”

**Review by chengtianwu — APPROVED** [review]

**Review by tonytonglt — APPROVED** [review]

**Review by chengtianwu — APPROVED** [review]

**Pull request merged**